app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -153,19 +153,24 @@ def rbac_tenant_edit
   end
 
   def tenant_form_fields
-    tenant = Tenant.find_by_id(params[:id])
+    tenant      = Tenant.find_by_id(params[:id])
+    root_tenant = tenant.default?
+    use_config  = tenant.use_config_for_attributes
 
     render :json => {
-      :name        => tenant.name,
-      :description => tenant.description,
-      :default     => tenant.default?,
-      :divisible   => tenant.divisible
+      :name                      => root_tenant && use_config ? MiqServer.my_server.get_config("vmdb").config.fetch_path(:server, :company) : tenant.name,
```

Comment:
note, if tenant has `root_tenant` and `use_config_for_attributes` set, then it will read out of the server settings - so I  don't think need this special logic here